web-portal/components/EventView.vue

```diff
@@ -20,12 +20,10 @@
         <!-- Online Resources technically have no fixed time, so suppress date display for them -->
         <div v-if="!isOnlineResource">
           <div v-for="(date_time, index) in event.date_times" :key="index" class="date-time-container">
-            <!-- TODO: do we need to consider timezones? If so, this might be useful: -->
-            <!-- https://stackoverflow.com/a/57022505 -->
             <!-- TODO: should we consider Luxon instead of Moment? -->
-            <em>{{ date_time.start_time | moment('dddd, MMMM Do') }}</em>
+            <em>{{ adjustDatetimeForTimezone(date_time.start_time, date_time.timezone).format('dddd, MMMM Do') }}</em>
             <br>
-            <em>{{ date_time.start_time | moment('h:mma') }} - {{ date_time.end_time | moment('h:mma') }}</em>
+            <em>{{ adjustDatetimeForTimezone(date_time.start_time, date_time.timezone).format('h:mma -') }} {{ adjustDatetimeForTimezone(date_time.end_time, date_time.timezone).format('h:mma z') }}</em>
```

Comment:
We should consider refactoring this to use computed properties, similar to the `when_date` and `when_time` in `EventCard`.